services/ingestion/poller/YrPoller/serverless.yml

```
@@ -22,7 +22,7 @@ provider:
   stackName: ${self:custom.stage}-${self:service}
   deploymentBucket:
     name: dataplattform-v2-deploymentbucket # Name of s3 bucket
-  runtime: python3.7
+  runtime: python3.8
```

Endrer du denne til 3.7 igjen for å ha det likt.
```suggestion
  runtime: python3.7
```